src/components/Paper.jsx

```diff
@@ -43,20 +45,27 @@ export function Papers() {
                 de apresentar o teu paper aos participantes da JORDI!
               </p>
             </div>
-            <form className="ml-36">
-              <h3 className="text-lg font-semibold tracking-tight text-blue-900">
-                Submete o Paper aqui{' '}
-                <span className="hidden sm:inline-flex" aria-hidden="true">
-                  &darr;
-                </span>
-              </h3>
-              <Button type="submit" className="ml-20 mt-2 sm:ml-24" href="/">
-                <span className="sr-only sm:not-sr-only">Submissão</span>
-                <span className="sm:hidden">
-                  <ArrowRightIcon className="h-6 w-6" />
-                </span>
-              </Button>
-            </form>
+            <div className="ml-36 flex-col gap-10">
+              <form>
+                <h3 className="text-lg font-semibold tracking-tight text-blue-900">
+                  Submete o Paper aqui{' '}
+                  <span className="hidden sm:inline-flex" aria-hidden="true">
+                    &darr;
+                  </span>
+                </h3>
+                <Button type="submit" className="ml-20 mt-2 sm:ml-24" href="/">
```

Comment:
Just my opinion, but I don't like the position of the button in mobile... I think it would be better after the text instead of below it